packages/desktop/src/lib.rs

```diff
@@ -213,12 +213,14 @@ pub fn launch_with_props<P: 'static + Send>(
             } => match event {
                 WindowEvent::CloseRequested => *control_flow = ControlFlow::Exit,
                 WindowEvent::Destroyed { .. } => desktop.close_window(window_id, control_flow),
-
-                WindowEvent::Resized(_) | WindowEvent::Moved(_) => {
-                    if let Some(view) = desktop.webviews.get_mut(&window_id) {
-                        let _ = view.resize();
-                    }
-                }
+                // WindowEvent::Resized(_) | WindowEvent::Moved(_) => {
+                //     if let Some(view) = desktop.webviews.get_mut(&window_id) {
+                //         let _ = view.resize();
+                //     }
+                // }
+                //
+                // No longer required manually resizing,
+                // see: https://github.com/tauri-apps/wry/commit/d7c9097256d76de7400032cf27acd7a1874da5cd
```

Comment:
that's cool! i think we can just delete these lines then, no need to leave them commented out

Reply:
[rMazeiks](https://github.com/rMazeiks) Ok

Reply:
@rMazeiks oh ok.